src/Explorer/Explorer.tsx

```diff
@@ -970,7 +1055,7 @@
~    useTabs.getState().activateNewTab(newTab);
~  }
~
~  public async openGallery(
~    selectedTab?: GalleryTabKind,
```

Comment:
>(userContext.features.notebooksTemporarilyDown === false && userContext.features.phoenix === true [](http://example.com/codeflow?start=7&length=97)

i am a little confused - is this ur proxy to detect if u r in github mode or temp storage mode?

Reply:
Thank you, missed this scenario[new notebook addition] when git logged in. Addressed.